.travis.yml

```diff
@@ -20,9 +20,10 @@ script:
   - "ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo"
 
   # Run the role/playbook again, checking to make sure it's idempotent.
+  # Changed is 1 as the packages database will update
   - >
     ansible-playbook -i tests/inventory tests/$PLAYBOOK --connection=local --sudo
-    | grep -q 'changed=0.*failed=0'
+    | grep -q 'changed=1.*failed=0'
```

Comment:
@Brunty ok so it seems this is one of those cases where, "it's not a bug, it's a feature" or sort of:
https://github.com/ansible/ansible-modules-core/issues/5484
http://docs.ansible.com/ansible/apt_module.html

A simple workaround would be to use the `cache_valid_time` flag, but, because SPR as discussed, I think I will later move such a task in a sort of `server-init` kind of role. I am going to test and merge this